docs/devel-quick-start.md

```diff
@@ -54,175 +49,73 @@ enough resources:
-different zones (e.g. US, Europe) and storage is replicated
-asynchronously. This environment includes 3 clusters - a hub cluster
-(`hub`), and 2 managed clusters (`dr1`, `dr2`) using `Ceph` storage.
-
-To start the environment, run:
+## Setting up the development environment
 
-```
-cd test
-drenv start envs/regional-dr.yaml
-```
+To set up a *Ramen* development environment you will need to run 3
+Kubernetes clusters. *Ramen* makes this easy using minikube, but you need
+enough resources:
 
-Starting takes at least 5 minutes, depending on your machine and
-internet connection.
+- Bare metal or virtual machine with nested virtualization enabled
+- 8 CPUs or more
+- 16 GiB of free memory
+- 20 GiB of free disk space
+- Internet connection
+- Linux - tested on *RHEL* 8.6 and *Fedora* 37.
 
-You can inspect the environment using kubectl, for example:
+You can run the make target `make create-rdr-env` to create a 3 cluster
+environment for suitable for Regional DR testing. This will create 3 minikube
+clusters, and install the necessary components for the Ramen operator to run.
+This uses `drenv` tool to setup the environment. Refer to
+[drenv readme](../test/README.md#setup) for more details.
 
-```
-kubectl get deploy -A --context hub
-kubectl get deploy -A --context dr1
-kubectl get deploy -A --context dr2
+```sh
+make create-rdr-env
 ```
 
 ## Building the ramen operator image
 
 Build the *Ramen* operator container image:
 
-```
+```sh
 make docker-build
 ```
 
 This builds the image `quay.io/ramendr/ramen-operator:latest`
 
 ## Deploying the ramen operator
 
-To deploy the *Ramen* operator in the test environment:
-
-```
-ramenctl deploy test/envs/regional-dr.yaml
-```
-
-## Configure the ramen operator on the hub
-
-Ramen need to be configured to know about the managed clusters and the
-s3 endpoints. To configure the ramen operator for test environment:
-
-```
-ramenctl config test/envs/regional-dr.yaml
-```
-
-For more info on the `ramenctl` tool see
+You can either deploy the *Ramen* operator using the make targets or use the
+`ramenctl` tool. For more info on the `ramenctl` tool see
 [ramenctl/README.md](../ramenctl/README.md).
 
-## Running system tests
-
-At this point *Ramen* is ready to protect workloads in your cluster, and
-you are ready for testing basic flows.
-
-To run basic tests using regional-dr environment run:
-
-```
-test/basic-test/run test/envs/regional-dr.yaml
-```
-
-This test:
+- Using `ramenctl`
+    - Deploy the *Ramen* operator in the test environment using `ramenctl`
 
-1. Deploys an application using
-   [ocm-ramen-samples repo](https://github.com/RamenDR/ocm-ramen-samples)
-1. Enables DR for the application
-1. Fails over the application to the other cluster
-1. Relocates the application back to the original cluster
-1. Disables DR for the application
-1. Undeploys the application
+     ```sh
+     ramenctl deploy test/envs/regional-dr.yaml
+     ```
```

Comment:
ramenctl comes from the virtual env, so we must create the env first (make venv) and enter the env before we run it.

The best way to run the virtual env is to port drenv and ramenctl to Go :-)

Reply:
Now the instructions remind that the virtual environment needs to be active.